embarkConfig/data.js

```diff
@@ -1,82 +1,106 @@
 /* eslint-disable complexity */
 const async = require('async');
 
-module.exports = async (GAS_PRICE, licensePrice, arbitrationLicensePrice, feeMilliPercent, burnAddress, mainnetOwner, fallbackArbitrator, deps) => {
+const estimateAndSend = (from, gasPrice) => async (toSend, value = 0) => {
+  const estimatedGas = await toSend.estimateGas({from, value});
+  return toSend.send({from, gasPrice, value, gas: estimatedGas + 10000});
```

Comment:
Isn't `+10000` a bit much? I don't mind, it's just that I think that 2000 is often enough

Reply:
It is too much!